### PR TITLE
Added Calico for Windows on AKS

### PR DESCRIPTION
**Pull request by song-jiang**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: windows-calico/limitations.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ canonical_url: '/getting-started/windows-calico/limitations'
 | **Feature** | **Not supported**                                            |
 | ----------- | ------------------------------------------------------------ |
 | Install     | Operator install                                             |
-|             | AKS, GKE, IKS, AWS, GCE, Azure, OpenShift, OpenStack, flannel |
+|             | GKE, IKS, OpenStack                                          |
 |             | K3 clusters                                                  |
 |             | Typha component for scaling (Linux-based feature)            |
 |             | ebpf (Linux-based feature)                                   |
```
Comment (bmckercher123) [inline windows-calico/limitations.md:15]: I believe the table has another error per our PR for EE. So not supported should say: Overlay mode with BGP peering. 

@song-jiang 
"Non-overlay mode with BGP peering. That is what we supported. We don't support Overlay mode with BGP peering."

File: windows-calico/quickstart.md [hunk 2]
```diff
@@ -48,7 +48,10 @@ Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Window
       kubectl delete rolebinding calico-install-token --namespace calico-system
       kubectl delete role calico-install-token --namespace calico-system
       ```
-
+- Additionally, for AKS:
+    - {{site.prodnameWindows}} can only be enabled on newly created clusters.
```
Comment (bmckercher123) [inline windows-calico/quickstart.md:52]: newly-created

File: windows-calico/quickstart.md [hunk 3]
```diff
@@ -321,6 +330,69 @@ The following steps install a Kubernetes cluster on a single Windows node, with
...
+   az feature register --namespace "Microsoft.ContainerService" --name "EnableAKSWindowsCalico"
+   ```
+
+1. Wait until `EnableAKSWindowsCalico` feature flag is registered successfully. Execute following cli command to get current status of the feature.
+
+   ```bash
+   az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/EnableAKSWindowsCalico')].{Name:name,State:properties.state}"
+   ```
+   
+   Move to next step if you see the same output from above command.
```
Comment (bmckercher123) [inline windows-calico/quickstart.md:348]: Move to next step if the output from above command matches the following output.

File: windows-calico/quickstart.md [hunk 4]
```diff
@@ -321,6 +330,69 @@ The following steps install a Kubernetes cluster on a single Windows node, with
...
+   -------------------------------------------------  ----------
+   Microsoft.ContainerService/EnableAKSWindowsCalico  Registered
+   ```   
+
+1. Refresh the registration of the `Microsoft.ContainerService` resource provider. Execute following command.
+
+   ```bash
+   az provider register --namespace Microsoft.ContainerService
+   ```
+
```
Comment (bmckercher123) [inline windows-calico/quickstart.md:360]: Create the AKS cluster with these settings: `network-plugin` to`azure`, and `network-policy` to `calico`.

File: windows-calico/quickstart.md [hunk 5]
```diff
@@ -48,7 +48,10 @@ Whether you use etcd or Kubernetes datastore (kdd), the datastore for the Window
       kubectl delete rolebinding calico-install-token --namespace calico-system
       kubectl delete role calico-install-token --namespace calico-system
       ```
-
+- Additionally, for AKS:
+    - {{site.prodnameWindows}} can only be enabled on newly created clusters.
+    - Only support Kubernetes version 1.20+
```
Comment (bmckercher123) [inline windows-calico/quickstart.md:53]: - Kubernetes version 1.20+

**Review by bmckercher123 — APPROVED** [review]
Minor changes.

File: windows-calico/quickstart.md [hunk 6]
```diff
@@ -321,6 +330,69 @@ The following steps install a Kubernetes cluster on a single Windows node, with
...
+    --vm-set-type VirtualMachineScaleSets \
+    --service-principal $your-service-principal \
+    --client-secret $your-client-secret \
+    --load-balancer-sku standard \
+    --node-vm-size Standard_D2s_v3 \
+    --network-plugin azure \
+    --network-policy calico
+   ```
+
+1. Add Windows node pool. For example,
```
Comment (lmm) [inline windows-calico/quickstart.md:382]: Add a Windows node pool

**Review by lmm — CHANGES_REQUESTED** [review]

**Comment by lmm** [comment]
Looking good so far, I added some minor comments. I guess while this could technically support Calico v3.17, in v3.17 we don't support k8s 1.20

File: windows-calico/quickstart.md [hunk 7]
```diff
@@ -321,6 +330,69 @@ The following steps install a Kubernetes cluster on a single Windows node, with
...
+1. Create AKS cluster with `network-plugin` set to `azure` and `network-policy` set to `calico`. For example,
+
+   ```bash
+   az group create -n $your-resource-group -l $your-region
+   az aks create \
+    --resource-group $your-resource-group \
+    --name $your-cluster-name \
+    --node-count 1 \
+    --enable-addons monitoring \
+    --windows-admin-username azureuser \
```
[the remark this answers is not in the document]
Comment (song-jiang) [reply windows-calico/quickstart.md:370]: Normally it is azureuser. It is in MSFT doc.

**Review by lmm — APPROVED** [review]

**Pull request merged**